code/datums/gamemodes/nuclear.dm

```diff
@@ -131,10 +131,14 @@
 	var/obj/landmark/closet_spawn = locate("landmark*Nuclear-Closet")
 
 	var/leader_title = pick("Czar", "Boss", "Commander", "Chief", "Kingpin", "Director", "Overlord", "General", "Warlord", "Commissar")
-	//var/agent_callsigns = list("Alpha", "Bravo", "Charlie", "Delta", "Echo", "Foxtrot", "Golf", "Hotel", "India", "Juliett", "Kilo", "Lima", "Mike", "November", "Oscar", "Papa", "Quebec", "Romeo", "Sierra", "Tango", "Uniform", "Victor", "Whiskey", "X-ray", "Yankee", "Zulu")
-	var/agent_callsigns = list("Axe", "Baselard", "Cutlass", "Dagger", "Estoc", "Falchion", "Gladius", "Hatchet", "Ice-pick", "Javelin", "Katana", "Longsword", "Machete", "Nodachi", "Odachi", "Partisan", "Quarterstaff", "Rapier", "Scimitar", "Tomahawk", "Uchigatana", "Voulge", "Warhammer", "Xiphos", "Yumi", "Zweihander")
 	var/leader_selected = 0
 
+	//Alphabetical agent callsign lists are delcared here, seperated in to catagories.
+	var/list/nato_callsigns = pick_string("agent_callsigns.txt", "nato")
+	var/list/melee_callsigns = pick_string("agent_callsigns.txt", "melee_weapons")
+	//Add new lists to the master callsign list.
+	var/list/callsign_list = pick(nato_callsigns, melee_callsigns)
```

Comment:
This should just be
```
var/list/callsign_list = strings("agent_callsigns.txt", "nato") + strings("agent_callsigns.txt", "melee_weapons")
```
then you can remove the `nato_callsigns` and `melee_callsigns` vars and as an added bonus the code will actually work!

(Currently nato_callsigns only gets assigned a *single* string instead of the whole list, that's what `pick_string` does as opposted to `strings`. And then `callsign_list` gets assigned either `nato_callsigns` or `melee_callsigns` instead of taking the two and merging them together, which can be achieved through the use of `+` instead.)

Reply:
Oh dang that looks much better, thanks!